sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClientLive/PhoneNumbersClientLiveTests.cs

```diff
@@ -46,14 +46,14 @@ public async Task ListPhoneNumbersAsync()
         [Test]
         [TestCase(null)]
         [TestCase("en-US")]
-        public async Task CreateReservationErrorState(string? locale)
+        public async Task CreateSearchErrorState(string? locale)
         {
             var client = CreateClient();
             const string countryCode = "US";
 
             // User and toll free is an invalid combination
             var searchOperation = await client.StartSearchAvailablePhoneNumbersAsync(countryCode, PhoneNumberType.TollFree, PhoneNumberAssignmentType.User,
-                new PhoneNumberCapabilities(PhoneNumberCapabilityValue.Outbound, PhoneNumberCapabilityValue.None));
+                new PhoneNumberCapabilities(PhoneNumberCapabilityValue.Outbound, PhoneNumberCapabilityValue.None), new PhoneNumberSearchOptions("212", 1));
```

Comment:
to improve readability let's use named parameters: `areaCode: "212", quantity: 1`

Reply:
Constructor is removed